dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
+        Custom Designs · Marketing · Branding · And More
+      </span>
+      <div class="whois">
+        <h2> WHO IS SEEK-A-DESIGN?</h2>
+        </br>
+        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
```

Comment:
Nice work. Now, can you:

- [Easy] - Add the remaining two paragraphs? (Make them each a separate `<p>` tag.
- [Harder] - Make it so that if the total text in this section doesn't fit inside its container, the text becomes scrollable.
    - Do this **without** allowing the title to scroll!